CMakeLists.txt

```diff
@@ -192,10 +194,14 @@ if (TBB_FIND_PACKAGE OR TBB_DIR)
     message(STATUS "Using external TBB for testing")
     find_package(TBB REQUIRED)
 else()
-    add_subdirectory(src/tbb)
+    if (TBB_BUILD)
+        add_subdirectory(src/tbb)
+    endif()
     if (NOT "${CMAKE_SYSTEM_PROCESSOR}" MATCHES "mips")
-        add_subdirectory(src/tbbmalloc)
-        add_subdirectory(src/tbbmalloc_proxy)
+        if (TBBMALLOC_BUILD)
+            add_subdirectory(src/tbbmalloc)
+            add_subdirectory(src/tbbmalloc_proxy)
+        endif()
         if (APPLE)
             message(STATUS "TBBBind build targets are disabled due to unsupported environment")
         else()
```

Comment:
There is `add_subdirectory(src/tbbbind)` one line below.

We've discussed that normally tbbbind is skipped due to lack of hwloc paths, but take into account auto-detection of hwloc using pkg-config: https://github.com/oneapi-src/oneTBB/blob/357c9b27fa84eaf748bc556585c26995c446eceb/cmake/hwloc_detection.cmake#L53

User still can force disabling hwloc detection through `TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH`. It's just FYI.